src/picker.js

```diff
@@ -281,6 +282,10 @@ angular.module("ion-datetime-picker", ["ionic"])
                     changeViewData();
                 };
 
+                $scope.timeFieldType = function() {
```

Comment:
Remove this and instead use the same approach as for example `mondayFirst` attr on line 311, in the link function.

```javascript
$scope.timeFieldType = "useTelType" in $attrs && $attrs.useTelType !== "false" ? "tel" : "text";
```

`use-tel-type` is shorter than `tel-type-for-time-fields` and IMHO still expresses its purpose.